### ISSUE TITLE
Return Empty data[]

### ISSUE THREAD

**Issue by Flow999Cloud** [Webklex/laravel-imap#130]
Hi, @Webklex 
Thanks for your great work

I am just trying to use this project on my web application
When I try to get the messages in the 'Inbox' folder, it returns empty data[] btw.
Could you explain me what I was wrong?

Here is a code snippet I tried.
```php
$oClient = new Client([
    'host'          => 'imap-mail.outlook.com',
    'port'          => 993,
    'encryption'    => 'ssl',
    'validate_cert' => true,
    'username'      => 'user@outlook.com',
    'password'      => 'secret',
]);
$oClient->connect();
$oFolder = $oClient->getFolder('Inbox');
$aMessage = $oFolder->query()->since('15.03.2018')->get();
var_dump($aMessage);
```

For now, $aMessage returns empty value: data[]

Thanks in advance for your help.

**Comment by Webklex** [comment]
Hi @Flow999Cloud ,
please take a look at #100 and perhaps the other [Outlook related issues](https://github.com/Webklex/laravel-imap/issues?q=Outlook). 
Outlook seems to be "special" sometimes...

Best regards

**Comment by Flow999Cloud** [comment]
Hi, @Webklex 

For update, I tried to use "Inbox" instead of "INBOX", but it doesn't work. Actually I have 335 emails in my email account. :)
For update2, I found all folder names using the above code snippet. I can see those are including Inbox, Archive, Trash etc. But when I try to use one of them, those are not working. :(

I'd like to try to use another php imap libraries for time being of the project.
I really appreciate for your help though. 

Thanks.
